src/components/Form/ContactUs.jsx

```diff
@@ -0,0 +1,197 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+
+export default function ContactUs() {
```

Comment:
It's awesome that you got this whole component do all the things like error and loading states. That is not trivial! 🥳  

The component uses a lot of separate state though. That's not wrong, but there is potential to simplify the code. If you're interested in doing so after you got your main user journey working (next week?), let me know and we can talk about how this could be potentially made more brief.